shared/gh/partials/admin-batch-edit.html

```diff
@@ -64,15 +64,15 @@ <h1 class="gh-jeditable-series-title"><%- data.series.displayName %></h1>
     </div>
 
     <div id="gh-batch-edit-term-container">
-        <% _.each(data.eventsByTerm, function(term, name) { %>
-            <div class="gh-batch-edit-events-container" data-term="<%- name.toLowerCase().replace(/\ /g, '') %>">
+        <% _.each(data.eventsByTerm, function(term) { %>
+            <div class="gh-batch-edit-events-container <% if (term.name == 'OT') { %>gh-ot<% } %>" data-term="<%- term.name.toLowerCase().replace(/\ /g, '') %>" data-start="<%- term.start %>" data-end="<%- term.end %>">
```

Comment:
I think we should improve here and not create a new table for each OT event but one per block of OT events.